File: 4_semester/DBMS/database-connections/csharp-connection/music-festivals/entities/Band.cs
namespace festival.model;

public class Band
{
    public int id { get; set; }
    public string name { get; set; }
    public string country { get; set; }
    public int formingYear { get; set; }

    public Band(string name, string country, int formingYear)
    {
        this.name = name;
        this.country = country;
        this.formingYear = formingYear;
    }
}

File: 4_semester/DBMS/concurrency-project/java-api/src/main/java/dbms/concurrency/backend/javaapi/Band.java
package dbms.concurrency.backend.javaapi;

import jakarta.persistence.*;
import lombok.*;

@NoArgsConstructor
@AllArgsConstructor
@Builder
@Data
@ToString
@Table(name = "bands_concurrency")
@Entity
public class Band {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
    private String name;
    private String country;
    private int forming_year;
}
